proto/role_binding/v1alpha1/role_binding.proto

```diff
@@ -0,0 +1,49 @@
+// Copyright 2025 Cofide Limited.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package proto.role_binding.v1alpha1;
+
+option go_package = "github.com/cofide/cofide-api-sdk/gen/go/proto/role_binding/v1alpha1";
+
+message RoleBinding {
+  string id = 1;
+  string role_id = 2;
+  oneof principal {
+    User user = 3;
+    Group group = 4;
+  }
+  Resource resource = 5;
+}
+
+message User {
+  // This is matched against the "sub" claim of the JWT presented by a user.
+  string id = 1;
+}
+
+message Group {
+  // For users this is matched against the list of values in the "groups" claim of the JWT presented by a user.
+  string id = 1;
```

Comment:
Again, would `name` be more appropriate?

Reply:
This claim is less well-defined (some implementations seem to populate it with values that look like UUIDs (Microsoft), others with names (Okta), and others allow users to define what goes in it).

Perhaps `claim_id`? As you noted with the User, want to avoid clashing with a potential future first-class entity, and both `id` and `name` seem like they would have specific meaning there?

Reply:
Agree that we shouldn't assume whether the group is a name or ID. `claim_value`?